File: src/main/java/model/CodeColor.java
package model;

public enum CodeColor {
    RED(255, 0, 0), YELLOW(255, 255, 0), GREEN(0, 255, 0);

    int r, g, b;

    CodeColor(int r, int g, int b) {
        this.r = r;
        this.g = g;
        this.b = b;
    }
}

File: src/main/java/model/CodeColor.kt
package model

enum class CodeColor(var r: Int, var g: Int, var b: Int) {
    RED(255, 0, 0), YELLOW(255, 255, 0), GREEN(0, 255, 0);

    override fun toString(): String = when(this) {
        RED -> "red"
        YELLOW -> "yellow"
        GREEN -> "green"
    }
}